app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/params/ValidConnectionMethodBase64EncodedDetails.scala

```diff
@@ -57,10 +103,43 @@ final case class Problem(message: String, percentage: Int) {
     } else {
       s"$percentage% of requests"
     }
+
+  val prettyCount: String = Problem.prettyCount(count)
+
 }
 
 object Problem {
   implicit val reads: Reads[Problem] = Json.reads[Problem]
+
+  private[tdq] def prettyCount(count: Int): String = {
+
+    val siPrefixes: Seq[String] = Seq("", "k", "m", "b")
+    val decimalFormat: DecimalFormat = new DecimalFormat("#.#")
+    val thousand: Double = 1000.0
+
+    def format(double: Double): String = {
+      val rounded = BigDecimal(double).setScale(1, BigDecimal.RoundingMode.FLOOR).toDouble
+      decimalFormat.format(rounded)
+    }
+
+    @tailrec def loop(d: Double, prefixes: Seq[String]): String =
+      if (prefixes.isEmpty) {
+        format(d)
+      } else if (d < thousand) {
+        if (d < 10) {
+          format(d) + prefixes.head
+        } else {
+          d.toInt + prefixes.head
+        }
+      } else {
+        loop(d / thousand, prefixes.tail)
+      }
```

Comment:
I was trying to see where `loop` is use and couldn't - the if statement on lines 137-142 looked like part of the method to me, a one line of space would be handy here

Reply:
Good spot! I tried to add `{ }` for the `loop` for clarity but scalafmt removes.

Added now.